README.md

```diff
@@ -162,6 +162,44 @@ Format your integration/script/playbook yml file according to Demisto's standard
 ` demisto-sdk format -t integration -p Integrations/Pwned-V2/Pwned-V2.yml`.
 This will go through the integration file, format it, and override the original file with the necessary changes.
 
+## Upload
+
+Upload integration to Demisto instance. DEMISTO_API_KEY environment variable should contain a valid Demisto API Key.
+
+**Arguments**:
+* *-i INTEGRATION_PATH, --infile INTEGRATION_PATH*
```

Comment:
I suggest to support also the package dir format. So if a directory is given the yml is generated to a temp file (as is done with unify) and uploaded. Saves the dev from running unify first.